modules/docker-build/main.tf

```diff
@@ -24,6 +24,7 @@ resource "docker_registry_image" "this" {
 resource "aws_ecr_repository" "this" {
   count = var.create_ecr_repo ? 1 : 0
 
+  force_delete         = true
```

Comment:
Please add `force_delete` variable with default value `true`.

Reply:
Added `var.ecr_force_delete` with a default value of `true`